packages/simcore-sdk/src/simcore_sdk/node_ports_common/_filemanager.py

```diff
@@ -41,6 +41,13 @@ async def _get_location_id_from_location_name(
     raise exceptions.S3InvalidStore(store)
 
 
+def _get_secure_link(url: AnyUrl) -> str:
```

Comment:
MINOR: this does not give you the secured link. It resolves the link correctly. Sometimes http and sometimes https, right?

Reply:
correct. reaamed